index.html

```diff
@@ -1,97 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 <head>
   <title>Classic LOGO</title>
-
-  <style>
-    * {
-      margin: 0;
-      box-sizing: border-box;
-    }
-    canvas {
-      outline: 1px solid gray;
-    }
-
-    .ui {
-      padding: 0px;
-      /* border: 1px solid gray; */
-    }
-    .commander {
-      padding: 5px;
-    }
-    .commander input, #history {
-      font-family: 'Lucida Console';
-      width: 100%;
-      padding: 10px;
-      margin-bottom: 5px;
-      border: 1px solid gray;
-      outline: none;
-      resize: vertical;
-      overflow-y: scroll;
-    }
-    .commander #history {
-      color: #414141;
-      height: 80px;
-      user-select: none;
-    }
-    .console {
-      display: flex;
-    }
-    .commander input {
-      flex: 1 1 1;
-      padding: 5px;
-      padding-left: 10px;
-      margin-right: 10px;
-      margin-bottom: 0;
-    }
-
-    button {
-      flex: 1;
-      flex-basis: 150px;
-      outline: 1px solid gray;
-      border: none;
-      background-color: white;
-      cursor: pointer;
-    }
-
-    button:hover {
-      background-color: #6b6b6b;
-      color: white;
-    }
-    button:active {
-      background-color: dimgray;
-      color: white;
-    }
-
-    #history p {
-      margin-top: 10px;
-      font-size: 12px;
-    }
-  </style>
+  <!--stylesheet-->
+  <link rel="stylesheet" href="./style/style.css">
 </head>
 <body>
-  
+
   <!-- <canvas id="c"></canvas> -->
 
   <div class="ui">
 
     <div class="commander">
       <div id="history" contenteditable="true"></div>
       <div class="console">
-        <input id="commands" value="repeat 150 [rt 15 fd repcount*2 rt 1 lt 9000/2]" spellcheck="false"/>
+        <input id="commands" value="repeat 150 [rt 15 fd repcount*2 rt 1 lt 9000/2]" spellcheck="false" />
         <button id="exeBtn">execute</button>
       </div>
     </div>
 
   </div>
-
-  <script src="./js/p5Vector.js"></script>
+  <script src="./script/p5Vector.js"></script>
   <!-- <script src="https://rawgit.com/anuraghazra/Candy.js/master/dist/Candy.build.js"></script> -->
   <!-- <script src="../src/lib/Candy.js"></script> -->
   <!-- <script src="./js/Candy.js"></script> -->
-  <script src="./js/p5min.js"></script>
-  <script src="./js/turtle.js"></script>
-  <script src="./js/index.js"></script>
+  <script src="./script/p5min.js"></script>
+  <script src="./script/turtle.js"></script>
```

Comment:
I prefer to have scripts in js folder not in scripts folder. So please change that.